lib/datadog/appsec/remote.rb

```diff
@@ -63,18 +65,20 @@ def receivers
             overrides = []
             exclusions = []
 
-            asm_data_config_types = ['blocked_ips', 'blocked_users']
-            asm_overrides_config_types = ['blocking', 'disabled_rules']
-
             repository.contents.each do |content|
               case content.path.product
               when 'ASM_DD'
                 rules << parse_content(content)
               when 'ASM_DATA'
-                data << parse_content(content) if asm_data_config_types.include?(content.path.config_id)
+                data << parse_content(content) if ASM_DATA_CONFIG_TYPES.include?(content.path.config_id)
               when 'ASM'
-                overrides << parse_content(content) if asm_overrides_config_types.include?(content.path.config_id)
-                exclusions << parse_content(content) if content.path.config_id == 'exclusion_filters'
+                parsed_content = parse_content(content)
+
+                if parsed_content['rules_override']
+                  overrides << parsed_content
+                else
+                  exclusions << parsed_content
+                end
```

Comment:
I feel like something's amiss here, I'm expecting something more like:

```
               if parsed_content['rules_override']
                  overrides['rules_override'] << parsed_content['rules_override']
                end
                if parsed_content['exclusions']
                  exclusions['exclusions'] << parsed_content['exclusions']
                end
```

Otherwise overrides end up including exclusions instead of merging them in exclusions.